### PR TITLE
URL UPDATED

### PR DESCRIPTION
**Pull request by PartishtaP**
THE CONTENTS i.e. THE NEW URL HAVE BEEN ADDED TO THE REQUIRED FILE

### CODE REVIEW

File: _data/partnerships.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 - name: "TalkPython Training"
   promo_code: "bpd-20pc-1c1da"
   description: "When you use the promo code: <strong>bpd-20pc-1c1da</strong> at checkout, you will receive 10% off your course purchase. TalkPython Training will also donate 20% of the purchase to Black Python Devs."
```
Comment (kjaymiller) [inline _data/partnerships.yml]:
```suggestion
  description: "When you use the <a href="https://training.talkpython.fm/black-python-devs">Black Python Devs landing page</a> or use the promo code: <strong>bpd-20pc-1c1da</strong> at checkout, you will receive 10% off your course purchase. TalkPython Training will also donate 20% of the purchase to Black Python Devs."
```

File: _data/partnerships.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
-  url: "/2024-08-16-talkpython-training-partnership"
+  url: "https://training.talkpython.fm/black-python-devs" # NEW URL UPDATED in _data/partnerships.yaml to now share affiliate landing page which provides an easier workflow for those that want to support Black Python Devs while also learning.
```
Comment (kjaymiller) [inline _data/partnerships.yml]:
```suggestion
  url: "https://training.talkpython.fm/black-python-devs" 
```

**Review by kjaymiller — CHANGES_REQUESTED** [review]
Thank you for this contribution. There was a little more that needs to be added to the issue but this looks good.

**Comment by PartishtaP** [comment]
If no changes are required can you please merge it and add my name as a contributor.

**Comment by kjaymiller** [comment]
> If no changes are required can you please merge it and add my name as a contributor.

There are changes requested which you can see in the PR.

File: _data/partnerships.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 - name: "TalkPython Training"
   promo_code: "bpd-20pc-1c1da"
-  description: "When you use the promo code: <strong>bpd-20pc-1c1da</strong> at checkout, you will receive 10% off your course purchase. TalkPython Training will also donate 20% of the purchase to Black Python Devs."
-  url: "/2024-08-16-talkpython-training-partnership"
+  description: "When you use the <a href="https://training.talkpython.fm/black-python-devs">Black Python Devs landing page</a> or use the promo code: <strong>bpd-20pc-1c1da</strong> at checkout, you will receive 10% off your course purchase. TalkPython Training will also donate 20% of the purchase to Black Python Devs."
```
Comment (kjaymiller) [inline _data/partnerships.yml]:
```suggestion
  description: 'When you use the <a href="https://training.talkpython.fm/black-python-devs">Black Python Devs landing page</a> or use the promo code: <strong>bpd-20pc-1c1da</strong> at checkout, you will receive 10% off your course purchase. TalkPython Training will also donate 20% of the purchase to Black Python Devs.'
```

**Review by kjaymiller — APPROVED** [review]

File: _data/partnerships.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 - name: "TalkPython Training"
   promo_code: "bpd-20pc-1c1da"
-  description: "When you use the promo code: <strong>bpd-20pc-1c1da</strong> at checkout, you will receive 10% off your course purchase. TalkPython Training will also donate 20% of the purchase to Black Python Devs."
-  url: "/2024-08-16-talkpython-training-partnership"
+  description: 'When you use the <a href="https://training.talkpython.fm/black-python-devs">Black Python Devs landing page</a> or use the promo code: <strong>bpd-20pc-1c1da</strong> at checkout, you will receive 10% off your course purchase. TalkPython Training will also donate 20% of the purchase to Black Python Devs.'
```
Comment (kjaymiller) [inline _data/partnerships.yml:3]:
```suggestion
  description: 'When you use the <a href="https://training.talkpython.fm/black-python-devs">Black Python Devs landing page</a> or use the promo code: <strong>bpd-20pc-1c1da</strong> at checkout, you will receive 10% off your course purchase. TalkPython Training will also donate 20% of the purchase to Black Python Devs.'

```

**Review by kjaymiller — CHANGES_REQUESTED** [review]

**Pull request merged**